scripts/hf_sim.py

```diff
@@ -189,6 +197,11 @@ def args_parser(cmd=None):
 
     args = parser.parse_args(cmd)
 
+    if args.site_specific is True:
+        args.hf_vel_mod_1d = None
+    else:
+        args.site_v1d_dir = None
```

Comment:
As all the locations testing site_v1d_dir should be testing args.site_specific, we don't need to erase these arguments?
We should probably add some checking for site_v1d_dir without site_specific too, as that wouldn't do anything
Ideally we'd also only take one of hf_vel_mod_1d and site_v1d_dir, but I think that's a bit too much of a refactor for right now

Reply:
Removed. Instead, added comment "ignored --site-specific is set" for hf_vel_mod_1d.